x-pack/plugins/observability/public/components/app/observability_status/content.ts

```diff
@@ -135,7 +135,7 @@ export const getContent = (
       addTitle: i18n.translate('xpack.observability.statusVisualization.alert.link', {
         defaultMessage: 'Create rules',
       }),
-      addLink: http.basePath.prepend('/app/management/insightsAndAlerting/triggersActions/rules'),
+      addLink: http.basePath.prepend('/app/observability/alerts'),
```

Comment:
@CoenWarmer we need to go to `/observability/alerts/rules` instead of `/observability/alerts`

Reply:
You're right of course!